### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by betula** [betula/use-between#20] [General]
Hey Guys! What do you think about this library, what positive or negative happens together!?
Please, share It, will be greatly welcome it!

**Comment by melloware** [comment]
@betula as I mentioned before this lib is awesome and it allowed me to simplify an app that was using Redux.  I was able to replace everything we were doing with Redux with just use-between and its tiny 2K footprint!

Plus personally I think the code is cleaner because with use-between it just looks like normal hooks and not anything special like Redux code.  I personally find it easier to read and understand than Redux!

**Reply by betula** [reply]
Thank you very much for your feedback! Cool! :blush: :+1:

**Reply by betula** [reply]
Hello @melloware, I will be grateful to you if you can tell how you use use-between, in which project and what tasks you solve!

**Reply by melloware** [reply]
I can't share the project as its private but the best use case we had is when a user logs into the app we create an `Account` object and that way everyone who needs that Account object on any page can just share it with a `use-between` hook!   Its a common use case people use Redux for.

**Reply by betula** [reply]
Many thanks!
